src/DocumentDB.ChangeFeedProcessor.UnitTests/PartitionManagement/PartitionControllerSplitTests.cs

```diff
@@ -15,211 +15,202 @@ namespace Microsoft.Azure.Documents.ChangeFeedProcessor.UnitTests.PartitionManag
     using Xunit;
 
     [Trait("Category", "Gated")]
-    public class PartitionControllerSplitTests : IAsyncLifetime
+    public class PartitionControllerSplitTests
     {
         private const string LastContinuationToken = "lastContinuation";
         private const string InitialContinuationToken = "initial token";
         private const string PartitionId = "partitionId";
-        private readonly ILease lease, leaseChild, leaseChild2;
-        private readonly ILeaseManager leaseManager;
-        private readonly IPartitionSupervisorFactory partitionSupervisorFactory;
-        private readonly IPartitionSynchronizer synchronizer;
-        private readonly PartitionController sut;
 
-        public PartitionControllerSplitTests()
+        private ILease MockLease(string partitionId = null)
```

Comment:
This should be named something like CreateMockLease.

Reply:
done